bookkeeper-server/pom.xml

```diff
@@ -222,8 +222,21 @@
         <version>2.7.3</version>
         <scope>test</scope>
     </dependency>          
+    <dependency>
+      <groupId>io.netty</groupId>
+      <artifactId>netty-tcnative-boringssl-static</artifactId>
+      <version>${netty-boringssl.version}</version>
+      <classifier>${os.detected.classifier}</classifier>
+    </dependency>
   </dependencies>
   <build>
+    <extensions>
+      <extension>
+        <groupId>kr.motd.maven</groupId>
```

Comment:
@fpj I think we will drop the plugin from this PR but we will need it for another issue, see #80 and my additional commit (that @sijie is going to append) https://github.com/eolivelli/bookkeeper/commit/4c7e571f48f5f7c713caa27d840b94f152f5b7ce 

it seems to me that it is licensed with Apache2 license